controllers/networking-calico/example/20-network.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Network
+metadata:
+  name: calico-network
+  namespace: default
+spec:
+  type: calico
+  podCIDR: 10.244.0.0/16
+  serviceCIDR:  10.96.0.0/24
+  providerConfig:
+    apiVersion: calico.extensions.gardener.cloud/v1alpha1
+    kind: NetworkConfig
```

Comment:
Please add at least `backend` to the example because it's not optional. You might also want to consider adding `IPAM` and `IPAutoDetectionMethod` (commented, as they are optional).